roadmap.md

```diff
@@ -94,15 +94,59 @@ Start receiving messages from the Ably service.
 
 ## Milestone 2: Realtime Connectivity Hardening
 
-Give users visibility of connection errors and enable the library to continue operating during tempoary loss of connection.
-
-- connection errors
-  - add the `DISCONNECTED` and `SUSPENDED` channel states
-  - handle connection opening errors ([`RTN14`](https://docs.ably.io/client-lib-development-guide/features/#RTN14))
-  - handle `DISCONNECTED` protocol messages ([`RTN15h`](https://docs.ably.io/client-lib-development-guide/features/#RTN15h))
-  - send resume requests ([`RTN15b`](https://docs.ably.io/client-lib-development-guide/features/#RTN15b))
-  - respond to connection resume responses ([`RTN15c`](https://docs.ably.io/client-lib-development-guide/features/#RTN15c))
-- fallbacks ([`RTN17`](https://docs.ably.io/client-lib-development-guide/features/#RTN17))
+This milestone will add connection error handling to the realtime client,
+allowing it to continue operating in the event of a recoverable connection error.
+It will also improve the visibility of what went wrong in the event of a fatal connection error.
+
+### Milestone 2a: Handle connection opening errors
+
+Implement the correct behaviour for all potential errors that may occur when establishing a new realtime connection.
+
+**Scope**:
+
+- Implement configurable `realtimeRequestTimeout` and transition to `DISCONNECTED` if the initial `CONNECTED` message is not received in time ([`RTN14c`](https://sdk.ably.com/builds/ably/specification/main/features/#RTN14c))
```

Comment:
`realtimeRequestTimeout` is used in a few other places (channel attach, authUrl/authCallback for realtime clients, etc) so I think the name is still appropriate. Maybe it would be beneficial to have separate configurable timeouts for each but yeah that would be a spec change 🤷